knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -219,7 +219,7 @@ func (r *ReconcileKnativeKafka) transform(manifest *mf.Manifest, instance *opera
 		setBootstrapServers(instance.Spec.Channel.BootstrapServers),
 		setAuthSecret(instance.Spec.Channel.AuthSecretNamespace, instance.Spec.Channel.AuthSecretName),
 		ImageTransform(common.BuildImageOverrideMapFromEnviron(os.Environ(), "KAFKA_IMAGE_"), log),
-		replicasTransform(manifest.Client),
+		replicasTransform(manifest.Client, &instance.Spec.HighAvailability.Replicas),
```

Comment:
Should we instead configure our own `KnativeKAfka` to be default `2` replicas if the HA is `nil`,

like here: https://github.com/openshift-knative/serverless-operator/blob/main/knative-operator/pkg/common/eventing.go#L18-L23

Reply:
added another function to do this